wiki/president_place_of_birth_tutorial.ipynb

```diff
@@ -0,0 +1,1638 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Extracting the Place of Birth of American Presidents from their Wikipedia HTML Pages"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you already have the database pob_presidents in your postgresql, please uncomment the first line to drop and create it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! dropdb --if-exists pob_presidents\n",
```

Comment:
The other tutorials have this line, but commented out with the thought that with it commented, a user's first run won't show a warning. You could change it to match, or I'm fine with this either way.